src/main/java/org/gridsuite/merge/orchestrator/server/repositories/MergeRepository.java

```diff
@@ -20,14 +21,31 @@
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com
  */
 @Repository
-public interface MergeRepository extends CassandraRepository<MergeEntity, MergeEntityKey> {
+public interface MergeRepository extends JpaRepository<MergeEntity, MergeEntityKey> {
 
-    @Query("SELECT * FROM merge WHERE processUuid = :processUuid")
-    List<MergeEntity> findByProcessUuid(@Param("processUuid") UUID processUuid);
-
-    @Query("DELETE FROM merge WHERE processUuid = :processUuid")
+    @Modifying
+    @Query(value = "DELETE from #{#entityName} as merge where merge.key.processUuid = :processUuid")
```

Comment:
Delete the specific query and use the standard Spring Data JPA with methods names

Reply:
Done in PR for migration to PostGres : https://github.com/gridsuite/merge-orchestrator-server/pull/50